pyproject.toml

```
@@ -13,16 +13,19 @@ dependencies = [
   "aiohttp-client-cache==0.11",
   "aiolimiter~=1.2",
   "aioresponses==0.7.6",
+  "arize-phoenix",
   "brotli",
   "esprima",
   "faiss-cpu==1.9.0",
   "gitpython",
   "google-search-results==2.4",
   "json5",
   "langchain>=0.3,<1.0",
+  "langchain-core>=0.3.80,<0.4.0",
+  "mcp>=1.0,<1.20",  # Avoid CVE-2025-45768: pyjwt added in 1.20.0
```

Minor suggestion to expand on comment
```suggestion
  "mcp>=1.0,<1.20",  # Avoid CVE-2025-45768: pyjwt added in 1.20.0. Remove when pyjwt CVE fix is released.
```